hacep/src/main/java/org/kie/hacep/Config.java

```diff
@@ -75,60 +75,50 @@ public static Properties getDefaultConfig() {
     }
 
     public static Properties getConsumerConfig(String caller) {
-        if(consumerConf == null){
+        if (consumerConf == null) {
             consumerConf = getDefaultConfigFromProps(CONSUMER_CONF);
         }
-        logConfig(caller,consumerConf);
+        logConfig(caller, consumerConf);
         return consumerConf;
     }
 
     public static Properties getProducerConfig(String caller) {
-        if(producerConf == null){
+        if (producerConf == null) {
             producerConf = getDefaultConfigFromProps(PRODUCER_CONF);
         }
-        logConfig(caller,producerConf);
+        logConfig(caller, producerConf);
         return producerConf;
     }
 
     public static Properties getSnapshotConsumerConfig() {
-        if(snapshotConsumerConf == null){
-            snapshotConsumerConf = getDefaultConfigFromProps(SNAPSHOT_CONSUMER_CONF);
+        if (snapshotConsumerConf == null) {
+            snapshotConsumerConf = getDefaultConfigFromProps(CONSUMER_CONF);
         }
-        logConfig("SnapshotConsumer",snapshotConsumerConf);
+        logConfig("SnapshotConsumer", snapshotConsumerConf);
         return snapshotConsumerConf;
     }
 
     public static Properties getSnapshotProducerConfig() {
-        if(snapshotProducerConf == null){
-            snapshotProducerConf = getDefaultConfigFromProps(SNAPSHOT_PRODUCER_CONF);
+        if (snapshotProducerConf == null) {
+            snapshotProducerConf = getDefaultConfigFromProps(PRODUCER_CONF);
         }
-        logConfig("SnapshotProducer",snapshotProducerConf);
+        logConfig("SnapshotProducer", snapshotProducerConf);
         return snapshotProducerConf;
     }
 
     public static Properties getDefaultConfigFromProps(String fileName) {
-            Properties config = new Properties();
-            InputStream in = null;
-            try {
-                in = Config.class.getClassLoader().getResourceAsStream(fileName);
-            } catch (Exception e) {
-            } finally {
-                try {
-                    config.load(in);
-                    in.close();
-                } catch (IOException ioe) {
-                    logger.error(ioe.getMessage(),
-                                 ioe);
-                }
-            }
-
-        if(config.get(BOOTSTRAP_SERVERS_KEY)== null){
-            config.put(BOOTSTRAP_SERVERS_KEY, getBootStrapServers());
+        Properties config = new Properties();
+        try (InputStream in = Config.class.getClassLoader().getResourceAsStream(fileName)) {
+            config.load(in);
+        } catch (Exception ex) {
+            logger.error(ex.getMessage(), ex);
```

Comment:
Is property file optional? If yes we can change log level from `error` to something like `info` or `debug`. If not we should throw an exception here

Reply:
The beloved `IllegalArgumentException` :)